plugin/vcs/vcs.go

```diff
@@ -22,6 +22,8 @@ const (
 	GitLabSelfHost Type = "GITLAB_SELF_HOST"
 	// GitHubCom is the VCS type for GitHub.com.
 	GitHubCom Type = "GITHUB_COM"
+	// SQLReviewAPISecretName is the api secret name used in GitHub action or GitLab CI workflow.
```

Comment:
```suggestion

		// SQLReviewAPISecretName is the api secret name used in GitHub action or GitLab CI workflow.
```

nit: use a new const or add an empty line above line 25 so that it differentiates from the two enums above.